pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -3474,3 +3479,15 @@ def get_additional_scorer_kwargs(self) -> Dict[str, Any]:
         """
         additional_scorer_kwargs = {"sp": self.sp_to_use}
         return additional_scorer_kwargs
+
+
+class TimeSeriesExperiment(TSForecastingExperiment):
+    def __init__(self) -> None:
+        msg = (
+            "DeprecationWarning: TimeSeriesExperiment class will be removed in "
+            "a future release. Please import the following instead. \n"
+            ">>> from pycaret.time_series import TSForecastingExperiment"
+        )
+        warnings.warn(msg, DeprecationWarning)
+        print(msg)
```

Comment:
can we log instead of print? the warning will be communicated to the user regardless.